scw/config.go

```diff
@@ -103,7 +103,7 @@ profiles:
 type Config struct {
 	Profile       `yaml:",inline"`
 	ActiveProfile *string             `yaml:"active_profile,omitempty"`
-	SendTelemetry bool                `yaml:"send_telemetry,omitempty"`
+	SendTelemetry *bool               `yaml:"send_telemetry,omitempty"`
```

Comment:
Does this enable telemetry by default?  So you have to explicitly opt-out?  I'm not a huge fan.
Are you sure this does not break config template generation?

Reply:
No, this does not enable telemetry by default. It should actually have no impact.

It's fixing an edge case where `send_telemetry: false` is the only thing written in the config and `config.Empty()` would return true.